std_msgs/Float64[] x_des
std_msgs/Float64[] u_ff
std_msgs/Float64MultiArray u_fb